src/common/Card/Card.tsx

```diff
@@ -1,9 +1,29 @@
 import React from "react";
-import { formatTime } from "./helper";
 import { Reminder } from "./Reminder";
 import * as UI from "./style";
 import PropTypes from "prop-types";
 import { ActivityCardProps } from "./interfaces";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowRight } from "@fortawesome/free-solid-svg-icons";
+import moment from "moment";
+
+/**
+ * Formats date to a string for the start and end time.
+ * @param time a date object
+ * @param duration a number in minutes
+ */
+export const formatTime = (time: Date, duration: number): string => {
+  /**
+   * returns string in HH:MM(AM/PM) format
+   * @param time a date object
+   */
+
+  return (
+    moment(time).format("MMMM Do h:mm a") +
+    " to " +
+    moment(new Date(time.getTime() + duration * 60000)).format("h:mm a")
```

Comment:
Moment.js provides us a nice helper function for this, we could use it and make this look a bit better:
``` js
moment().add(duration, "m").format("h:mm a")
```
[Reference](https://momentjs.com/docs/#/manipulating/add/)

Reply:
i actually had some questions about how we were planning on displaying this part based on how we were storing the data in the yaml files, so I'll fix this altogether after the next meeting :+1: